runtime/stdlib/flow.go

```diff
@@ -47,7 +47,7 @@ var accountFunctionType = &sema.FunctionType{
 	),
 }
 
-var getAccountFunctionType = &sema.FunctionType{
+var GetAccountFunctionType = &sema.FunctionType{
```

Comment:
Looks like this change is not necessary anymore

Reply:
yeah right, missed reverting this. good catch!